backend/src/main/java/org/atlantbh/internship/auctionapp/service/api/StripeService.java

```diff
@@ -0,0 +1,19 @@
+package org.atlantbh.internship.auctionapp.service.api;
+
+import com.stripe.exception.StripeException;
+import com.stripe.model.Customer;
+import com.stripe.model.PaymentMethod;
+import org.atlantbh.internship.auctionapp.entity.CreditCardEntity;
+import org.atlantbh.internship.auctionapp.exception.BadRequestException;
+import org.atlantbh.internship.auctionapp.request.CreateCustomerRequest;
+
+import java.util.List;
+
+public interface StripeService {
```

Comment:
This should be named `PaymentService` and `StripeServiceImpl` should be named `StripeService`